packages/frontend/src/utils/localStorage.ts

```diff
@@ -0,0 +1,20 @@
+import { captureException } from '@sentry/react';
+
+export const getFromLocalStorage = <T>(key: string): T | null => {
+    try {
+        const item = localStorage.getItem(key);
```

Comment:
> In the AppProvider, we clear out that localStorage so we always start with a fresh state

Is this not the same as using sessionStorage ?

Reply:
I had to look this up last week 😂 SessionStorage only clears if the tab or window is closed. Otherwise, the state remains. LocalStorage persists unless explicitly cleared.